src/app/oxford-debate/page.tsx

```diff
@@ -63,20 +63,26 @@ export default function OxfordDebate() {
   const debateconfig = useLang("oxfordDebateConfiguration");
 
   const conf = useContext(DebateContext).conf;
-  const [adVocemAudio, stateAdVocemAudio, controlAdVocemAudio] = useAudio({
-    src: conf.soundPack.adVocemSound || "",
-  });
-  const [debateEndAudio, stateDebateEndAudio, controlDebateEndAudio] = useAudio(
-    {
-      src: conf.soundPack.debateEndSound || "",
-    }
-  );
+
+  const advocemSound = ((res) => ({
+    element: res[0],
+    state: res[1],
+    controls: res[2],
+  }))(useAudio({ src: conf.soundPack.adVocemSound || "" }));
+
+  const debateEndSound = ((res) => ({
+    element: res[0],
+    state: res[1],
+    controls: res[2],
+  }))(useAudio({ src: conf.soundPack.debateEndSound || "" }));
```

Comment:
I like the idea of an object with element, state, and controls – that's way clearer. But what is `res`? It's impossible to infer its form or purpose it from this fragment alone.

Reply:
Its only purpose is to feed into the object, and as such there is no need for its form to be inferred by the programmer. It's the exact same type as the tuple that was in its place before - just with different names. It is never interacted with again, as it's only within scope of the function that contructs the object. Naming these temporary elements, and having to substitute "res" for "[temporaryAdVocemSoundElement, temporaryAdVocemSoundState, temporaryAdVocemSoundControls] and "res[0|1|2]" for each of the instances of the tuple members produces code that is needlessly verbose and takes up way too much space for what it does (which is nothing). ((On a side note, I am not even completely satisfied with the syntax used here, but I could find no way that takes up less space.))

The name "res" is a common abbreviation of "result" - it is a tuple that holds the resulting values you get when calling useAudio.

Reply:
`temp` would be better. It says "you don't have to care about it" way clearer than `res`.